README.md

```diff
@@ -2,7 +2,7 @@
 
 # Project Name
 
-> Description the project.
+> This is the Hello Prject
```

Comment:
- [ ] Good job using the correct readme template 👏 to make your readme file more professional kindly add more description to your readme file according to the existing sections,  example : 

1. Getting Started: tell the user how to start with your project and clone it locally
2. Setup & Install: describe to the user how to set up your project and install it locally 
3. etc ...

- [ ] to make your readme file clean remove the sections that you don't need for this project example: the live demo section, etc ...
- [ ] in the authors section kindly make sure you add your correct social media links instead of the example ones
- [ ] your licence link is currently broken , kindly fix it

Reply:
Thankyou @ichala for your comments.
I have resolved the issues as per your recommendations. 👍